packages/formation/sass/base/_va.scss

```diff
@@ -400,6 +400,10 @@ article > h1 {
     padding-bottom: 0.5em;
   }
 
+  h2 {
```

Comment:
This is a pretty broad change that I'd want to look at applied to vets-website before we merge it here. I think it may affect some paragraph spacing in content pages.

Reply:
I ran a search on `vagov-content` and looked at 12 pages of resuluts to see if any h2s are in use in a `feature` component. That does not appear to be the case. Mostly, I'm seeing h3 as the top of level heading inside a `feature`, which may not always be semantically correct. On some of the pages, the headings skip from h1 -> h3.

Reply:
wonder if @1Copenut might have thoughts on this.

Reply:
Oh, I actually misread the code here. Didn't realize it was nested. (_shakes fist at sass nesting_)

Reply:
_also shakes fist at sass nesting_